tests/client/core/TestAddonManager.js

```diff
@@ -36,6 +36,16 @@ describe('addonManager', () => {
     };
   });
 
+  describe('hasAddonManager', () => {
+    it('does if mozAddonManager is in navigator', () => {
+      assert.ok(addonManager.hasAddonManager({ navigator: { mozAddonManager: {} } }));
+    });
+
+    it('does not if mozAddonManager is not in navigator', () => {
+      assert.notOk(addonManager.hasAddonManager({}));
```

Comment:
calling this like `addonManager.hasAddonManager()` would prevent someone from accidentally removing the param initializer.